FirebaseTestingSupport/Functions/Sources/Public/FirebaseFunctionsTestingSupport/FIRFunctions+Internal.h

```diff
@@ -0,0 +1,72 @@
+// Copyright 2017 Google
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This file is a copy of Functions/Internal/FIRFunctions+Internal.h except for the modular import
+// of FirebaseFunctions.
+
+#import <Foundation/Foundation.h>
+
+@import FirebaseFunctions;
+
+@protocol FIRAppCheckInterop;
+@protocol FIRAuthInterop;
+@protocol FIRMessagingInterop;
+@class FIRHTTPSCallableResult;
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface FIRFunctions (Internal)
```

Comment:
optional: Not sure if the extension names can actually conflict, but I would use a unique name just in case (e.g. `Testing` instead of `Internal`).